CMake/External_qtExtensions.cmake

```diff
@@ -34,6 +34,7 @@ ExternalProject_Add(qtExtensions
   CMAKE_ARGS
     ${COMMON_CMAKE_ARGS}
     ${QT_ARGS}
+    -DCMAKE_INSTALL_RPATH:PATH=${fletch_INSTALL_BUILD_DIR}/lib
```

Comment:
...not `fletch_INSTALL_DIR`? (Or is that not a thing?) Not saying this is wrong, just somewhat unexpected. (Also, I guess we are sure that `lib` is always correct on macOS? Should this be macOS-specific?)

Reply:
`fletch_INSTALL_BUILD_DIR` is what is used to set `fletch_ROOT` in the exports, so I think this is what I want.  I believe fletch is configured to use `lib` on all platforms.

Reply:
That's correct, `fletch_INSTALL_BUILD_DIR` is used to set `fletch_ROOT`. That said, I'm not sure why we have that variable at all. It's just getting set to the `fletch_BUILD_INSTALL_PREFIX` which is what's passed in to change the location of the installed subprojects. In this case either will work, but I am thinking it might be best just to remove `fletch_INSTALL_BUILD_DIR` at some point so everything is consistent.